packages/shared/providers.yaml

```diff
@@ -4665,6 +4665,22 @@ typeform:
         base_url: https://api.typeform.com
     docs: https://docs.nango.dev/integrations/all/typeform
 
+typefully:
+    display_name: Typefully
+    categories:
+        - twitter
```

Comment:
Please correct the categories to ;
- analytics
- communication
- social

Also add to their corresponding docs pages
`<Card title="Typefully" href="/integrations/all/typefully" color="#68a063" />`